### PR TITLE
Fix case comments pagination limit

### PR DESCRIPTION
**Pull request by cristianonicolai**
Review page size to -1 instead of Integer.MAX_VALUE

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch @jhrcek please have a look.

File: jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/main/java/org/jbpm/console/ng/cm/backend/server/RemoteCaseManagementServiceImpl.java [hunk 1]
```diff
@@ -47,7 +47,7 @@
 
     @Override
     public List<CaseDefinitionSummary> getCaseDefinitions() {
-        final List<CaseDefinition> caseDefinitions = client.getCaseDefinitions(0, Integer.MAX_VALUE);
+        final List<CaseDefinition> caseDefinitions = client.getCaseDefinitions(0, -1);
```
Comment (jhrcek) [inline jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/main/java/org/jbpm/console/ng/cm/backend/server/RemoteCaseManagementServiceImpl.java]: What does page of size `-1` mean? I'd prefer named constat, like `PAGE_SIZE_UNLIMITED` to make it clear.

**Comment by jhrcek** [comment]
It's not clear to me what does page size -1 mean, alghough I susptec it means "retrieve everything till the end". Could you extract it as named constant to clarify the meaninig? Like `PAGE_SIZE_UNLIMITED`

**Comment by jhrcek** [comment]
+1

**Pull request merged**